.azure-pipelines/pipelines.yml

```diff
@@ -548,10 +548,22 @@ stages:
       condition: always()
 
   - job: docker
+    strategy:
+      matrix:
+        windows2019:
+          imageName: 'windows-latest'
```

Comment:
It might be better to use `windows-2019` here so it is not affected when azp update its latest image.